CMakeLists.txt

```diff
@@ -45,6 +51,7 @@ message(STATUS "C++ compiler flags: ${CMAKE_CXX_FLAGS}")
 if (SNOWHOUSE_BUILD_TESTS)
     FILE(GLOB SnowhouseSpecSourceFiles example/*.cpp)
     add_executable(snowhouse-tests ${SnowhouseSpecSourceFiles})
+    target_link_libraries(snowhouse-tests PRIVATE snowhouse)
```

Comment:
Does that work with old CMake versions? (I am not sure if we should actively support CMake pre-3.0 versions. I guess you added the support because of the failing CI... If it is easy to support it (as above), we should.)